Firmware/communication/can/can_simple.hpp

```diff
@@ -35,6 +35,8 @@ class CANSimple {
         MSG_SET_LINEAR_COUNT,
         MSG_SET_POS_GAIN,
         MSG_SET_VEL_GAINS,
+        MSG_GET_ADC_VOLTAGE,
+        MSG_SEND_ADC_VOLTAGE,
```

Comment:
this value is not used anywhere

Reply:
My use case is reading voltage from a potentiometer that measures absolute output position of a robotic joint to be used when homing or detecting minimum and maximum joint angles

I updated my files to remove the extra MSG_SEND_ADC_VOLTAGE enum so ill update the pull request to reflect that. I only use MSG_GET_ADC_VOLTAGE for both sending and receiving data

Signing CLA tonight

Reply:
Okay, just realized I have no idea how to update the pull request. Have to work on it tomorrow

Reply:
You can do a fresh checkout of the `master` branch and then cherry-pick your commit on top of that and then force-push.

Reply:
Hmm, comparing the two they look the same. MSG_GET_ADC_VOLTAGE is used to kick off the correct callback function and MSG_SEND_ADC_VOLTAGE is used to set the txmsg.id inside the callback so the return message has a different id than the send message. So both the variables are used. If you think having a single enum would be better I can change it.

This is my first PR and I haven't used GitHub since so sorry about the confusion.

Reply:
On which line is MSG_SEND_ADC_VOLTAGE used? I don't see it anywhere in the code.

Regarding the git stuff, just make sure the number X in the tab "Commits (X)" shows "1", so your branch is based on our `devel` plus all your changes in one single commit. See my previous message on how to achieve that. But if it sounds too complicated could also just do a new clone and copy over the changes.

Reply:
On line 353 of can_simple.cpp. I left a comment above